packages/create-web-scripts-library/README.md

```diff
@@ -0,0 +1,35 @@
+# `create-web-scripts-library`
+
+Scaffold a [@spotify/web-scripts](https://github.com/spotify/web-scripts) library quickly. Code was directly inspired by [create-next-app](https://github.com/zeit/create-next-app).
+
+## Usage
+
+### With `yarn create`
+
+```sh
+yarn create @spotify/web-scripts-library my-cool-library
```

Comment:
yarn somehow knows where to inject the `create`?

Reply:
yep! https://github.com/yarnpkg/yarn/pull/6239